domains/bruce.json

```diff
@@ -0,0 +1,13 @@
+{
+  "description": "bruce is a dev",
+  "repo": "https://github.com/brucewangyq",
+  "owner": {
+    "email": "brucewangyq@gmail.com",
+    "twitter": "brucewangdev",
+    "username": "brucewangyq"
+  },
+  "record": {
+    "TXT": "vc-domain-verify=bruce.is-a.dev,a33081fa59e0ed06cc4b",
```

Comment:
Sorry, we don't support TXT verification.

Reply:
Also, we don't support Vercel.

Reply:
Thank you for your reply, I changed to CNAME.